ddl/ddl.go

```diff
@@ -383,9 +383,14 @@ func (d *ddl) GetLease() time.Duration {
 	return lease
 }
 
-// GetInformationSchema get the infoschema binding to d. It's expoted for testing.
-func (d *ddl) GetInformationSchema() infoschema.InfoSchema {
-	return d.infoHandle.Get()
```

Comment:
Some DDL actions are in use `d.infoHandle.Get()`, no use `GetInformationSchema` will have what problem?
#### For example
* `AddIndex`
* `AddColumn`
* `DropColumn`

Reply:
@ciscoxll 
No.
In "GetInformationSchema" I add the hook that uses to test. So it's OK.

Reply:
Got it.